test/es-module/test-http-imports-cli.mjs

```diff
@@ -0,0 +1,48 @@
+import { mustCall } from '../common/index.mjs';
+import { match, notStrictEqual } from 'assert';
+import { spawn } from 'child_process';
+import { execPath } from 'process';
+
+{
+  const child = spawn(execPath, [
+    '--experimental-network-imports',
+    '--input-type=module',
+    '-e',
+    'import "http://example.com"',
+  ]);
+
+  let stderr = '';
+  child.stderr.setEncoding('utf8');
+  child.stderr.on('data', (data) => {
+    stderr += data;
+  });
+  child.on('close', mustCall((code, _signal) => {
+    notStrictEqual(code, 0);
+
+    // [ERR_NETWORK_IMPORT_DISALLOWED]: import of 'http://example.com/' by
+    //   …/[eval1] is not supported: http can only be used to load local
+    // resources (use https instead).
+    match(stderr, /[ERR_NETWORK_IMPORT_DISALLOWED]/);
```

Comment:
Nit: I prefer the `err.code` & `err.message.includes()` used in other ESM error tests. I think consistency would be favourable.

Reply:
We don't have access to `err.code` not `err.message`, we're reading the stderr of a subprocess. FYI this test was inspired from https://github.com/nodejs/node/blob/15f1a45c5f52fbf3d35389428846a5c86d684a87/test/es-module/test-esm-import-json-named-export.mjs#L7-L24

Reply:
gah, yes indeed.